pkgs/development/ocaml-modules/bitstring/default.nix

```diff
@@ -1,14 +1,14 @@
-{ stdenv, fetchzip, buildOcaml, time, autoconf, automake }:
+{ stdenv, fetchurl, buildOcaml, time, autoconf, automake }:
 
 buildOcaml rec {
   name = "bitstring";
-  version = "f1673f8"; 
-  src = fetchzip {
-    url = https://storage.googleapis.com/google-code-archive-source/v2/code.google.com/bitstring/source-archive.zip;
-    sha256 = "03343yggwp3y483zj5axaalxlnl698xrjiv3hmd4c2s05g53iwas";
+  version = "2.1.0";
+  src = fetchurl {
+    url = http://github.com/xguerin/bitstring/archive/v2.1.0.tar.gz;
+    sha256 = "0miw4banfpmx4kxrckpqr57b1fcmsqdmspyjx6gqjd4kghm4l7xj";
   };
 
-  patches = [ ./camlp4-git.patch ./meta.patch ./srcdir.patch ];
+  patches = [ ./camlp4-git.patch ];
 
```

Comment:
Can you tell me the environment in which this fails? It builds locally for me, and I'd like to know how to observe the build failure to avoid this in the future. (I've added srcdir.patch back in locally, and it'll be in a changed PR in a few minutes after I finish doing a clean rebase on master.)